### PR TITLE
Verbosity

### LINKED ISSUE #347: Provide mechanism for greater verbosity
Currently the "verbose" option is a flag. Ansible (following the lead of ssh) provides for a count of up to 4 levels of verbosity by increasing the number of times the flag is present (e.g. greater verbosity can be had by passing "-v", "-vv", "-vvv", and "-vvvv").

Improve linchpin to permit passing multiple -v options, which would be passed through to the underlying Ansible calls to increase its verbosity.

### PR DESCRIPTION
**Pull request by herlo**
This PR fixes #347 and adds up to -vvvv on the linchpin call, passing it through to the Ansible playbooks for output debugging.

### CODE REVIEW

**Comment by p3ck** [comment]
[test] Hopefully trigger a jenkins build

**Comment by herlo** [comment]
[test] rerun jenkins tests

**Comment by herlo** [comment]
[test] trigger another build

File: linchpin/ansible_runner.py [hunk 1]
```diff
@@ -3,19 +3,21 @@
 import ansible
 from collections import namedtuple
 from contextlib import contextmanager
-from callbacks import PlaybookCallback
 
 ansible24 = float(ansible.__version__[0:3]) >= 2.4
 
 
 # CentOS 6 EPEL provides an alternate Jinja2 package
+# whichi is used by the imports below
```
Comment (p3ck) [inline linchpin/ansible_runner.py]: nit - typo

which is used by the imports below

**Pull request merged**